docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-issues/vm.md

```diff
@@ -63,7 +62,7 @@ do
 done
 ```
 
-## Firewall enabled for nodes
+## Firewall is enabled for nodes
 
 You can add firewall exceptions on the nodes with private IP addresses `10.1.13.150`, `10.1.13.151`, and `10.1.13.152` by executing the following command:
```

Comment:
why am i doing this?

Reply:
I think I fixed it, I'll get dev to review to make sure it's correct.